### DISCUSSION TITLE
Welcome to kotlin-logging Discussions!

### DISCUSSION THREAD

**Discussion by oshai** [MicroUtils/kotlin-logging#253] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vasilMihaylovHub** [comment]
Hello,

I migrated my project to kotlin-logging and encountered some issues. I'm wondering if these are already known.

   1. Wrong line number and method invocation in the log. We don't intend to migrate all of our logs, so we need it to work like this:
   logger.info("Starting..")
   However, when we call it this way in the log, we see the line number of the info method:
   public fun info(msg: String?): Unit = info { msg }
   with this pattern: 
   <Pattern>%M.%L : %msg%n</Pattern>       
   no matter from where it is called it write this in the log: 
   info.518 : Starting..     
   
   2. Missing logs: The issue here is that methods not being implemented:
   public fun info(msg: String?, arg: Any?): Unit = TODO()
   public fun info(msg: String?, arg1: Any?, arg2: Any?): Unit = TODO()
  
   
Can you assist me in resolving these problems?

**Reply by oshai** [reply]
which version of kotlin-logging are you using?

**Reply by vasilMihaylovHub** [reply]
1.9.22

**Reply by oshai** [reply]
please use latest kotlin-logging (6.0.3 at the moment)

**Reply by vasilMihaylovHub** [reply]
I am using the latest version.

**Reply by vasilMihaylovHub** [reply]
Do you think it is a good idea to create an issue for these problems?

**Reply by oshai** [reply]
Can you please share a repository to reproduce this?

**Reply by vasilMihaylovHub** [reply]
Here is the issue with the incorrect row number. Unfortunately, I could not reproduce the problem with the missing log.
https://github.com/vasilMihaylovHub/dummy-kotlin.git

**Reply by oshai** [reply]
sorry for the delay. try using:
```
kLogger.info {" Row number test" } 
```
 ( info {} instead of deprecated info()). 
 tried that and it worked.

**Reply by vasilMihaylovHub** [reply]
Thank you. For now, migrating everywhere from info() to info{} is impossible, so my team and I will continue using slf4j.

Thank you for the prompt communication, and I wish you all the best.
